lib/ui/painting.dart

```diff
@@ -928,9 +928,20 @@ Future<Codec> instantiateImageCodec(Uint8List list) {
 String _instantiateImageCodec(Uint8List list, _Callback<Codec> callback)
   native "instantiateImageCodec";
 
-/// Convert an image file from a byte array into an [Image] object.
-void decodeImageFromList(Uint8List list, ImageDecoderCallback callback)
-    native "decodeImageFromList";
+/// Loads a single image frame from a byte array into an [Image] object.
+///
+/// This is a convenience wrapper around [instantiateImageCodec].
+/// Prefer using [instantiateImageCodec] which also supports multi frame images.
+// We only keep this method for backward compatibility.
```

Comment:
I would remove this comment. If you think about it, everything we support is something we support for backwards compatibility. :-)